### DISCUSSION TITLE
Creating Tensors - I don'T see tensor(7) as Daniel does

### DISCUSSION THREAD

**Discussion by denijazz84** [mrdbourke/pytorch-deep-learning#1013] [Q&A] [answered]
Dears,
I just started learning. I am typing 
# scalar
scalar = torch.tensor(7)
scalar

but it gives me this ;
tensor([ 8.4078e-45,  0.0000e+00, -1.7698e-26,  3.1494e-41, -1.7699e-26,
         3.1494e-41,  0.0000e+00])

instead of this;

scalar = torch.Tensor(7)

Where do I do mistake ?
Thank you !

**Comment by denijazz84** [comment, marked as answer]
Ok I got it, I wrote T big in torch.Tensor(7), but it has to be small letter, torch.tensor(7)

**Comment by ghost** [comment]
Here’s why ```torch.Tensor(7)``` didn't work as expected:

```torch.Tensor()``` (uppercase) without arguments creates a tensor of size 0, hence the unexpected output you saw (tensor([ ... ])).
torch.tensor() (lowercase) with a numerical argument creates a scalar tensor with that value.
So, to summarize, use torch.tensor() (lowercase) for creating scalar tensors with specific values.
```py
import torch

scalar = torch.tensor(7)
print(scalar)

```
